src/ray/raylet/node_manager.cc

```diff
@@ -1030,7 +992,7 @@ void NodeManager::ResourceUsageAdded(const NodeID &node_id,
   }
 }
 
-void NodeManager::ResourceUsageBatchAdded(
+void NodeManager::ResourceUsageBatchReceived(
```

Comment:
How about rename it to `UpdateBatchResourceUsage`?

Reply:
This is a callback to handle notification of batched resource usage, so it's better to used `Received`